pydm/widgets/timeplot.py

```diff
@@ -348,9 +770,59 @@ def channels(self):
                          BasePlot.setMaxYRange, doc="""
     Maximum Y-axis value visible on the plot.""")
 
+    def getShowRightAxis(self):
```

Comment:
It looks like there is no way for users to actually do anything with the right axis.  Are there plans to add configurable properties for it in the future?

We eventually need to add right axis support to the WaveformPlot as well, so I'd appreciate it if you can do as much of this in BasePlot as possible, so that it is easy to reuse it in the other plot types.

Reply:
Fixed the toggling problem of the right y-axis.

Moved this setting to BasePlot.

@hhslepicka, I believe the right y-axis can be set to display a different set of y-values, at a different scale. In addition, usually we just hide the right y-axis in a chart, but the left y-axis is always present. So, I don't think we need to worry about hiding the left y-axis. Instead, maybe we can think about the next step as to make the right y-axis be able to display values at a different scale.